### DISCUSSION TITLE
The 2024 PR is `Ready for Review`  [Repositories Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#146926] [Repositories]
![Header for the Repositories category shows a dark purple abstract background with white text reading “Repositories Check-in”. In the foreground there is an image of a person holding a magnifying glass next to a cartoon representation of a group of repositories. To the right, there is a GitHub logo in two-tone purple.](https://github.com/community/community/assets/144172738/06f2261b-e8f4-40a3-b92b-99cf29b86134)

With the end-of-year holidays in full swing, this check-in has plenty more to celebrate— from top solves of the year, to recent releases, and exciting community happenings. 

## Wall of Fame 🤩 : These folks make our category so great ! Here are some of the top-answered questions from 2024 🎉
* [How to get tabs in my project readme section](https://github.com/orgs/community/discussions/86658): @rockyMM broke down the “why” here and included a snazzy, easy to read chart - thanks for solving this top question! 
* [How to give admin access to collaborators on personal repository](https://github.com/orgs/community/discussions/108527): @muizzhaque chimed in and shared plenty of documentation for a thorough response. This discussion was viewed over 8,000 times multiplying the impact of your contribution. 
* [How to only make one specific branch of a repository accessible to the public](https://github.com/orgs/community/discussions/131435): This discussion drew quite the crowd, but @davevad93’s answer, which included step-by-step directions led the original poster to victory and @davevad93 even continued to assist the original poster through some back-and-forth! 

## 🗳️ Repository Releases: All updates, info, releases, etc that are Repo-related!
* **Pull request merge method rule – Public Preview**: Repository rules now allow you to enforce which merge methods are available when merging pull requests into a specified branch. The merge method rule is available for rulesets at the repository, organization and the enterprise level. Learn more and share your thoughts in [this discussion thread](https://github.com/orgs/community/discussions/146284). 
* **Enterprise repository properties, policies and rulesets – Public Preview**: This preview includes enterprise custom repository properties, enterprise repository policies and enterprise rulesets to help enterprise administrators manage more at greater scale. Share questions, feedback, and comments in [our discussion](https://github.com/orgs/community/discussions/145599). 

## 📣 Announcements: The latest updates around the Community and GitHub 🆕
* **❗GitHub Incident Updates**: As of December 5, incidents will now be posted in the community. [Read our announcement](https://github.com/orgs/community/discussions/146469). 
* **Git your learning on with GitHub Copilot Certification**: We’re in our last week of the four-week study guide to help you prepare. [Join us in this Discussion](https://github.com/orgs/community/discussions/144443) and the course to explore essential AI topics, engage in discussions, and try hands-on exercises that’ll make you exam-ready. 💚
* **The top 10 gifts for the developer in your life**: Whether you’re hunting for the perfect gift for your significant other, the colleague you drew in the office gift exchange, or maybe (just maybe) even for yourself, @laviniasf has you covered with our top 10 gifts that any developer would love in her [blogpost](https://github.blog/news-insights/company-news/the-top-10-gifts-for-the-developer-in-your-life/). 

With gratitude being a key theme during this time of the year, I want to extend a heartfelt thanks to everyone in the Repos category whether you left a single comment confirming you were also experiencing a bug or you consistently answered questions around the category (I’m looking at you @davevad93) thanks for being here and making this category so great ♥️.

**Comment by davevad93** [comment]
Thanks for the mention @queenofcorgis , I'm very happy to be an active member of the Community, love y'all! ❤️

**Reply by queenofcorgis** [reply]
Thanks for being such a vital part of our community! Happy holidays if you celebrate ✨

**Comment by RockyMM** [comment]
Well, shucks, I never expected this 😊 Thanks for the mention @queenofcorgis; it was a pleasure to share something with others which puzzled me for a while too :)

**Reply by queenofcorgis** [reply]
@RockyMM that is so awesome, thanks for the comment - happy 2025!

**Comment by queenofcorgis** [comment]
Check out who is already making an impact in our category in 2025 with [our check-in just posted in a new discussion](https://github.com/orgs/community/discussions/149743)!

**Comment by LaviniaSF** [comment]
Thanks for the shout out! 🎉
